recipe/meta.yaml

```diff
@@ -7,48 +12,48 @@ package:
 source:
   fn: mpfr-{{ version }}.tar.bz2
   url: http://ftp.gnu.org/gnu/mpfr/mpfr-{{ version }}.tar.gz
-  sha256: ae26cace63a498f07047a784cd3b0e4d010b44d2b193bab82af693de57a19a78
-  patches:
-  - patches/build-vc14.patch
-  - patches/corei5.patch
+  sha256: 116715552bd966c85b417c424db1bbdf639f53836eb361549d1f8d6ded5cb4c6
 
 build:
-  number: 1
-  skip: True  # [win and vc<14]
+  number: 3
   run_exports:
     - {{ pin_subpackage("mpfr") }}
+  missing_dso_whitelist:
+    - $RPATH/ld64.so.1  # [s390x]
 
 requirements:
   build:
     - libtool         # [unix]
     - m4              # [unix]
```

Comment:
Given both variants call `build.sh` which runs `configure`, I'm surprised that you don't need `{{ posix }}libtool` or `{{ posix }}m4` on `win-64` (ie. no selector like `{{ posix }}make` below).  Or, flipping that on its head, maybe they're not required on `unix` either?
Maybe `msys2-libtool` and `msys2-m4` are pulled in transitively?

Reply:
libtool is invoked a lot but not there as a conda package at all. for linux, it's there as a conda package.
there's no mention of m4 in the log. For linux, it isn't used.
So m4 can be removed, but I don't know which libtool is being used.

Reply:
removing both means compilation is ok on win and linux.
it uses at build time a `./libtool` script in the root directory which is generated automatically by config.status which is generated by configure.
So it seems like mpfr wants to use its own libtool and m4 rather than taking from external, which seems fine and probably not a good idea to try and un-vendor (hard and might break things)?

Reply:
there is also an /m4 directory which is populated at some point during the build, I guess during the configure step.